scalafmt-tests/src/test/resources/scala3/InlineTransparent.stat

```diff
@@ -0,0 +1,22 @@
+maxColumn = 20
+<<< simple inline
+inline val logging =     false
+>>>
+inline val logging = false
+<<< complex inline
+inline val logging = {
+    if (a) { true}
+    else false
+}
+>>>
+inline val logging = {
+  if (a) { true }
+  else false
+}
+<<< inline transparent
+transparent inline def choose(b: Boolean): A = {if(b) {new A} else {new B}}
+>>>
+transparent inline def choose(b: Boolean): A = {
+  if (b) { new A }
+  else { new B }
+}
```

Comment:
eol

Reply:
Added!

Reply:
i see still a "do not enter" sign :)

Reply:
Damn, I really thought I changed it! Hopefully now it's fine.